packages/types/src/json.ts

```diff
@@ -736,3 +738,21 @@ export interface CommerceCheckoutJSON extends ClerkResourceJSON {
   totals: CommerceCheckoutTotalsJSON;
   is_immediate_plan_change: boolean;
 }
+
+export interface ApiKeyJSON extends ClerkResourceJSON {
+  id: string;
+  type: string;
+  name: string;
+  subject: string;
+  scopes: string[];
+  claims: Record<string, any> | null;
+  revoked: boolean;
+  revocation_reason: string | null;
+  expired: boolean;
+  expiration: number | null;
```

Comment:
We usually do
```suggestion
  expires_at: number | null;
```
right ?

Reply:
It's `expiration` [in bapi proxy](https://github.com/clerk/cloudflare-workers/blob/ce638d632c49593987ee3f6fe267ef56d1e4adb9/workers/bapi-proxy/src/routers/api-keys/serialize.ts#L29) 👀